### ISSUE TITLE
fPath issue in output of tut 1 on cloudina

### ISSUE THREAD

**Issue by Bergylta** [ocean-data-factory-sweden/kso#383]
## 🐛 Bug
There seem to be a issue with how the fpath is written in the output file in the cloudina CSV file for the movies

## To Reproduce (REQUIRED)

Input:
```
990805-Yttre-Vattenholmen.mp4
```

Output:
```
/buckets/kso/output/1999_CN/990805-Yttre-Vattenholmen.mp4
```

## Expected behavior
Behaviour should be that the fpath is the same as movie file name if mp4, else if movie is not mp4, the only difference should be that fpath is "moviename".mp4

## Environment

 - OS: [e.g. Ubuntu]
 - GPU [e.g. 2080 Ti]

**Comment by jannesgg** [comment]
Fixed by d0268d7a0514a2aaa83b89c5346e4be53bc166d5

**Comment by jannesgg** [comment]
Issue persists and the fpath should be adjusted somewhere else.
